SPECS/hdf5/hdf5.spec

```diff
@@ -0,0 +1,1054 @@
+%global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
+# No more Java on i686
+%ifarch %{java_arches}
+%bcond_without java
+%else
+%bcond_with java
+%endif
```

Comment:
We always have Java, so we can drop these `bcond_with*` macros and keep the parts of the spec where `%{with java}` evaluates to `true`.